specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2021-02-01-preview/dataprotection.json

```diff
@@ -4364,6 +4703,19 @@
         }
       }
     },
+    "SyncBackupInstanceRequest": {
+      "description": "Sync BackupInstance Request",
+      "required": [
+        "forceResync"
+      ],
+      "type": "object",
+      "properties": {
+        "forceResync": {
+          "description": "Specifies if resync needs to be done forced otherwise only resync will happen for failure case.",
+          "type": "boolean"
```

Comment:
>boolean [](http://example.com/codeflow?start=19&length=7)

ARM generally recommends against booleans in favor of string enums. Best option is to choose a better property (e.g. syncType: { "ForceResync", "LazySync", "Default" }). This type of property is easier to understand and use. Next best option if you really want a True/False value is to use a string enum "forceResync": { "True", "False" }. Both options allow adding values later without requiring a new API version.

Reply:
Sure.. will fix.

Reply:
Fixed. Thanks.